src/drivers/imu/mpu9250/mpu9250.cpp

```diff
@@ -383,7 +383,9 @@ MPU9250::read_reg(unsigned reg, uint32_t speed)
 {
 	uint8_t buf{};
 
-	_interface->read(MPU9250_SET_SPEED(reg, speed), &buf, 1);
+	if (_interface->read(MPU9250_SET_SPEED(reg, speed), &buf, 1) < 0) {
+		PX4_WARN("MPU9250::read_reg failed");
```

Comment:
```suggestion
		PX4_DEBUG("MPU9250::read_reg failed");
```

Reply:
If this actually fails we can't have it spamming the console continuously.

Reply:
I know that, but I'm not going to let this pr being merged so that is only for test. If this kind of hack can be committed back to upstream, I would like to open up another pr.